src/index.scss

```diff
@@ -0,0 +1,14 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: #060b35;
+}
+
+@import './styles/variables';
+@import './styles/header';
+@import './styles/navigation';
+@import './styles/card';
+@import './styles/stars';
```

Comment:
Ensure that the file './styles/stars' exists and is correctly referenced. SCSS imports require the correct path to the file.